domain/domain.go

```diff
@@ -1004,7 +1008,8 @@ func recoverInDomain(funcName string, quit bool) {
 		return
 	}
 	buf := util.GetStack()
-	log.Errorf("%s, %v, %s", funcName, r, buf)
+	logutil.Logger(context.Background()).Error("recover in domain failed", zap.String("function_name", funcName),
```

Comment:
s/function_name/funcName.
As stated in the documentation, if it is a variable, then we use the way of the hump.